lib/core-complete.sh

```diff
@@ -1661,6 +1683,8 @@ function ble/complete/util/eval-pathname-expansion {
   IFS= GLOBIGNORE= builtin eval "ret=(); ret=($pattern)" 2>/dev/null
 
   ble/util/invoke-hook dtor
+
+  ble/complete/source/limit ret
```

Comment:
Can you change to exit with non-zero status keeping the results of the pathname expansion, and checks the exit status at the caller side to let the caller decide whether to cancel the processing or not?

Reply:
ok!